viewing_party/party.py

```diff
@@ -1,23 +1,156 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if (not title) or (not genre) or (not rating):
+        return None
+    
+    movie_dict = {"title": title, "genre": genre, "rating": rating}
+    return movie_dict
+
+def add_to_watched(user_data, movie):
+    new_watched_list = user_data["watched"]
+    new_watched_list.append(movie)
+    user_data["watched"] = new_watched_list
+    return user_data

```

Comment:
Check out what happens when you remove line 13. If you run it, the tests still pass, so what's up with that? Well, lists are passed by reference, meaning that whether we append using the variable `user_data["watched"]` or `new_watched_list`, the same list is being appended. We don't have to re-assign `user_data["watched"]` since it is referencing the same list that was changed.

Otherwise, this implementation works fine! But it is a little wordy. We could do this in 1 or 2 lines without creating new variables.

```suggestion
def add_to_watched(user_data, movie):
    user_data["watched"].append(movie)
    return user_data

```